### ISSUE TITLE
Popup select list value not announced by screen readers

### ISSUE THREAD

**Issue by stefandobre** [tinymce/tinymce#8729]
<img width="495" alt="menu-button" src="https://github.com/tinymce/tinymce/assets/20504443/ddd39687-04c7-42ef-b3cf-f2d4287f6176">

The TinyMCE poups contain a menu button control, that really should be a standard HTML select list:
- Insert / Edit Link dialog: 'Open link in..' field
- Table properties > Advanced > Border style
- possibly others

When navigating through the fields, they are announced as menu buttons, they are labelled by the correct label (eg 'open link in...'), however you don't hear the value unless you open the menu and see which value is checked. Ideally, this should just be implemented as a standard HTML select list.

Tested with Voiceover on MacOS and Jaws 2023 on Windows.

**Comment by TheSpyder** [comment]
We are actually moving more of our dialogs away from HTML `select` elements as those are very difficult to apply styles to. We do have a tracking ticket for various accessibility issues at the moment so hopefully this will be resolved soon.
